### DISCUSSION TITLE
How to change Mysense screan?

### DISCUSSION THREAD

**Discussion by albochl** [cedricp/ddt4all#1076] [Q&A]
Hello. I have question. I have new Renault Arkana. (Korea name XM3, LJL)
Currently, my MultiSense screen is set to an XM3 image, and I want to change this XM3 image to a Renault Arkana image, but I can't find the path. I currently have the latest ecu database from here, but I can't find XJL. I want to know how to change it. Any help would be greatly appreciated. Please understand that I do not speak English so I am using a translator.

**Comment by matzel82** [comment]
Do you came from Germany?
